### ISSUE TITLE
ranger crashed when trying to delete files 

### ISSUE THREAD

**Issue by lakshyabijarnia** [ranger/ranger#2300]
#### Runtime Environment

- Operating system and version: ubuntu 20.04
- Terminal emulator and version: GNOME Terminal version 3.36.2 A terminal emulator for the GNOME desktop
    Using VTE version 0.60.3 +BIDI +GNUTLS +ICU +SYSTEMD
- Python version: 3.8.5 (default, Jan 27 2021, 15:41:15) [GCC 9.3.0]
- Ranger version/commit: ranger 1.9.3
- Locale: en_IN.ISO8859-1

#### Context

I was trying to trash two files with help of ranger but as soon as i confirmed and entered y in the confirmation space it crashed and the file did not get trashed

#### Steps to reproduce
1. open ranger in terminal 
2. select any file for trash
3. enter y in the conformation space.

#### Traceback

```File "/usr/lib/python3/dist-packages/ranger/core/main.py", line 201, in main
    fm.loop()
  File "/usr/lib/python3/dist-packages/ranger/core/fm.py", line 383, in loop
    ui.handle_input()
  File "/usr/lib/python3/dist-packages/ranger/gui/ui.py", line 266, in handle_input
    self.handle_key(key)
  File "/usr/lib/python3/dist-packages/ranger/gui/ui.py", line 196, in handle_key
    elif not DisplayableContainer.press(self, key):
  File "/usr/lib/python3/dist-packages/ranger/gui/displayable.py", line 275, in press
    focused_obj.press(key)
  File "/usr/lib/python3/dist-packages/ranger/gui/widgets/console.py", line 188, in press
    self.type_key(key)
  File "/usr/lib/python3/dist-packages/ranger/gui/widgets/console.py", line 214, in type_key
    self._answer_question(answer)
  File "/usr/lib/python3/dist-packages/ranger/gui/widgets/console.py", line 197, in _answer_question
    callback(answer)
  File "/usr/lib/python3/dist-packages/ranger/config/commands.py", line 759, in _question_callback
    self.fm.execute_file(files, label='trash')
  File "/usr/lib/python3/dist-packages/ranger/core/actions.py", line 459, in execute_file
    filenames = [f.path for f in files]
  File "/usr/lib/python3/dist-packages/ranger/core/actions.py", line 459, in <listcomp>
    filenames = [f.path for f in files]
AttributeError: 'str' object has no attribute 'path'

ranger crashed.

```

**Comment by toonn** [comment]
Are you sure this was with delete and not trash? Trash had problems that have recently been fixed on master.

**Comment by lakshyabijarnia** [comment]
sorry this was problem wit trash

**Comment by toonn** [comment]
Then I'll go ahead and close this because that issue was fixed in #1871.
